2024-11-16 15:31:38,011 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-16 15:31:38,011 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Configure stats pid to 40186
2024-11-16 15:31:38,012 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/.config/wandb/settings
2024-11-16 15:31:38,012 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/settings
2024-11-16 15:31:38,012 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-16 15:31:38,012 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-16 15:31:38,012 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-16 15:31:38,013 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-16 15:31:38,013 INFO    MainThread:40186 [wandb_init.py:_log_setup():533] Logging user logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/run-20241116_153138-ftjnc1wo/logs/debug.log
2024-11-16 15:31:38,013 INFO    MainThread:40186 [wandb_init.py:_log_setup():534] Logging internal logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/run-20241116_153138-ftjnc1wo/logs/debug-internal.log
2024-11-16 15:31:38,014 INFO    MainThread:40186 [wandb_init.py:_jupyter_setup():479] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x73d747013070>
2024-11-16 15:31:38,015 INFO    MainThread:40186 [wandb_init.py:init():619] calling init triggers
2024-11-16 15:31:38,016 INFO    MainThread:40186 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'dataset': 'StrongPassword.csv', 'model': 'XGBoost', 'param_grid': {'n_estimators': [50, 100], 'max_depth': [3, 5], 'learning_rate': [0.1, 0.2]}, 'test_size': 0.2}
2024-11-16 15:31:38,016 INFO    MainThread:40186 [wandb_init.py:init():669] starting backend
2024-11-16 15:31:38,016 INFO    MainThread:40186 [wandb_init.py:init():673] sending inform_init request
2024-11-16 15:31:38,021 INFO    MainThread:40186 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-16 15:31:38,022 INFO    MainThread:40186 [wandb_init.py:init():686] backend started and connected
2024-11-16 15:31:38,034 INFO    MainThread:40186 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2024-11-16 15:31:38,037 INFO    MainThread:40186 [wandb_init.py:init():781] updated telemetry
2024-11-16 15:31:38,045 INFO    MainThread:40186 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-16 15:31:38,892 INFO    MainThread:40186 [wandb_init.py:init():867] starting run threads in backend
2024-11-16 15:31:39,302 INFO    MainThread:40186 [wandb_run.py:_console_start():2456] atexit reg
2024-11-16 15:31:39,302 INFO    MainThread:40186 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2024-11-16 15:31:39,303 INFO    MainThread:40186 [wandb_run.py:_redirect():2370] Wrapping output streams.
2024-11-16 15:31:39,303 INFO    MainThread:40186 [wandb_run.py:_redirect():2395] Redirects installed.
2024-11-16 15:31:39,306 INFO    MainThread:40186 [wandb_init.py:init():911] run started, returning control to user process
2024-11-16 15:32:19,243 INFO    MainThread:40186 [wandb_run.py:_finish():2155] finishing run hanifrobby-brawijaya-university/Keystroke Dynamics/ftjnc1wo
2024-11-16 15:32:19,243 INFO    MainThread:40186 [jupyter.py:save_history():459] not saving jupyter history
2024-11-16 15:32:19,244 INFO    MainThread:40186 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-11-16 15:32:19,244 INFO    MainThread:40186 [wandb_init.py:_jupyter_teardown():461] cleaning up jupyter logic
2024-11-16 15:32:19,244 INFO    MainThread:40186 [wandb_run.py:_atexit_cleanup():2420] got exitcode: 0
2024-11-16 15:32:19,244 INFO    MainThread:40186 [wandb_run.py:_restore():2402] restore
2024-11-16 15:32:19,244 INFO    MainThread:40186 [wandb_run.py:_restore():2408] restore done
2024-11-16 15:36:29,065 INFO    MainThread:40186 [wandb_settings.py:_apply_login():1989] Applying login settings: {}
2024-11-16 15:36:29,074 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-16 15:36:29,074 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Configure stats pid to 40186
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/.config/wandb/settings
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/settings
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-16 15:36:29,075 INFO    MainThread:40186 [wandb_init.py:_log_setup():533] Logging user logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/run-20241116_153629-30esuyjy/logs/debug.log
2024-11-16 15:36:29,076 INFO    MainThread:40186 [wandb_init.py:_log_setup():534] Logging internal logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/run-20241116_153629-30esuyjy/logs/debug-internal.log
2024-11-16 15:36:29,076 INFO    MainThread:40186 [wandb_init.py:_jupyter_setup():479] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x73d7ac2ed060>
2024-11-16 15:36:29,076 INFO    MainThread:40186 [wandb_init.py:init():619] calling init triggers
2024-11-16 15:36:29,076 INFO    MainThread:40186 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'dataset': 'StrongPassword.csv', 'model': 'XGBoost', 'param_grid': {'n_estimators': [50, 100], 'max_depth': [3, 5], 'learning_rate': [0.1, 0.2]}, 'test_size': 0.2}
2024-11-16 15:36:29,077 INFO    MainThread:40186 [wandb_init.py:init():669] starting backend
2024-11-16 15:36:29,077 INFO    MainThread:40186 [wandb_init.py:init():673] sending inform_init request
2024-11-16 15:36:29,079 INFO    MainThread:40186 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-16 15:36:29,080 INFO    MainThread:40186 [wandb_init.py:init():686] backend started and connected
2024-11-16 15:36:29,085 INFO    MainThread:40186 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2024-11-16 15:36:29,086 INFO    MainThread:40186 [wandb_init.py:init():781] updated telemetry
2024-11-16 15:36:29,087 INFO    MainThread:40186 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-16 15:37:59,115 ERROR   MainThread:40186 [wandb_init.py:init():841] encountered error: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-11-16 15:37:59,115 ERROR   MainThread:40186 [wandb_init.py:init():1266] error in wandb.init()
Traceback (most recent call last):
  File "/home/crystallizer/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
  File "/home/crystallizer/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 847, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-11-16 15:38:47,828 INFO    MainThread:40186 [wandb_settings.py:_apply_login():1989] Applying login settings: {}
2024-11-16 15:38:47,832 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-16 15:38:47,832 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Configure stats pid to 40186
2024-11-16 15:38:47,832 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/.config/wandb/settings
2024-11-16 15:38:47,832 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/settings
2024-11-16 15:38:47,832 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-16 15:38:47,832 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_init.py:_log_setup():533] Logging user logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/run-20241116_153847-ovhs8csa/logs/debug.log
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_init.py:_log_setup():534] Logging internal logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/run-20241116_153847-ovhs8csa/logs/debug-internal.log
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_init.py:init():619] calling init triggers
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'dataset': 'StrongPassword.csv', 'model': 'XGBoost', 'param_grid': {'n_estimators': [50, 100], 'max_depth': [3, 5], 'learning_rate': [0.1, 0.2]}, 'test_size': 0.2}
2024-11-16 15:38:47,833 INFO    MainThread:40186 [wandb_init.py:init():669] starting backend
2024-11-16 15:38:47,834 INFO    MainThread:40186 [wandb_init.py:init():673] sending inform_init request
2024-11-16 15:38:47,835 INFO    MainThread:40186 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-16 15:38:47,835 INFO    MainThread:40186 [wandb_init.py:init():686] backend started and connected
2024-11-16 15:38:47,838 INFO    MainThread:40186 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2024-11-16 15:38:47,839 INFO    MainThread:40186 [wandb_init.py:init():781] updated telemetry
2024-11-16 15:38:47,840 INFO    MainThread:40186 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-16 15:40:17,858 ERROR   MainThread:40186 [wandb_init.py:init():841] encountered error: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-11-16 15:40:17,858 ERROR   MainThread:40186 [wandb_init.py:init():1266] error in wandb.init()
Traceback (most recent call last):
  File "/home/crystallizer/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
  File "/home/crystallizer/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 847, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-11-16 15:41:56,219 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-11-16 15:41:56,220 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Configure stats pid to 40186
2024-11-16 15:41:56,220 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/.config/wandb/settings
2024-11-16 15:41:56,220 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/settings
2024-11-16 15:41:56,220 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-11-16 15:41:56,220 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-16 15:41:56,220 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_init.py:_log_setup():533] Logging user logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/offline-run-20241116_154156-4lxx0tmy/logs/debug.log
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_init.py:_log_setup():534] Logging internal logs to /home/crystallizer/Documents/Skripsi/Code/Documentation/wandb/offline-run-20241116_154156-4lxx0tmy/logs/debug-internal.log
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_init.py:init():619] calling init triggers
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'dataset': 'StrongPassword.csv', 'model': 'XGBoost', 'param_grid': {'n_estimators': [50, 100], 'max_depth': [3, 5], 'learning_rate': [0.1, 0.2]}, 'test_size': 0.2}
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_init.py:init():669] starting backend
2024-11-16 15:41:56,221 INFO    MainThread:40186 [wandb_init.py:init():673] sending inform_init request
2024-11-16 15:41:56,223 INFO    MainThread:40186 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-16 15:41:56,224 INFO    MainThread:40186 [wandb_init.py:init():686] backend started and connected
2024-11-16 15:41:56,226 INFO    MainThread:40186 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2024-11-16 15:41:56,227 INFO    MainThread:40186 [wandb_init.py:init():781] updated telemetry
2024-11-16 15:41:56,228 INFO    MainThread:40186 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-11-16 15:43:26,248 ERROR   MainThread:40186 [wandb_init.py:init():841] encountered error: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-11-16 15:43:26,248 ERROR   MainThread:40186 [wandb_init.py:init():1266] error in wandb.init()
Traceback (most recent call last):
  File "/home/crystallizer/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
  File "/home/crystallizer/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 847, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
